content/dstu2/binary.md

```diff
@@ -36,4 +36,49 @@ Retrieving the Binary resource with its native mime type
 <%= headers 200, 'Content-Type': 'text/html' %>
 <%= html(:dstu2_binary_html) %>
 
+## Operation autogen-ccd-if
```

Comment:
This looks a little strange when rendered (moreso for docref). Can we update these to something like:
```
## Operation: autogen-ccd-if
or
## $autogen-ccd-if Operation
```